src/collections/Events.ts

```diff
@@ -1,5 +1,7 @@
 import type { CollectionConfig, PayloadRequest } from 'payload/types';
-import { Project, Event, EventsMedia, Guild } from '../payload-types';
+import {
+	Project, Event, EventsMedia, Guild,
```

Comment:
```suggestion
	Project, Event, EventMedia, Guild,
```
You also still need to regenerate the type file.